JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -92,14 +93,15 @@ module Microsoft.ApplicationInsights {
                 this.session = new Context.Session();
                 this.sample = new Context.Sample(config.sampleRate());
             }
+
+            this._addDefaultTelemetryInitializers();
```

Comment:
Can you remove telemetry initializer? Or without strong types in JS you'd never know which one to remove?